sdk/java/pulumi/src/main/java/com/pulumi/context/internal/ConfigContextInternal.java

```diff
@@ -3,31 +3,29 @@
 import com.pulumi.Config;
 import com.pulumi.context.ConfigContext;
 import com.pulumi.core.internal.annotations.InternalUse;
+import com.pulumi.internal.ConfigInternal;
 
 import javax.annotation.ParametersAreNonnullByDefault;
-import java.util.function.Function;
 
 import static java.util.Objects.requireNonNull;
 
 @InternalUse
 @ParametersAreNonnullByDefault
 public class ConfigContextInternal implements ConfigContext {
```

Comment:
we can simplify this class now, the original implementation was a workaround in the anticipation of the changes done in this PR